terraform/datadog/cloud_controller.tf

```diff
@@ -114,10 +114,10 @@ resource "datadog_monitor" "cc_job_queue_length" {
   escalation_message  = "Job queue in Cloud Controller API still too big, check the API health."
   require_full_window = false
 
-  query = "${format("max(last_30m):max:cf.cc.job_queue_length.total{deployment:%s} > 10", var.env)}"
+  query = "${format("avg(last_30m):max:cf.cc.job_queue_length.total{deployment:%s} > 10", var.env)}"
```

Comment:
Also this is the graph in CI for rolling average of 5m: 

![screenshot from 2016-12-20 16 49 15](https://cloud.githubusercontent.com/assets/280032/21359500/5d5e570a-c6d4-11e6-9518-787f13867420.png)

Said that, I could set 5m and set a threshold of 18-20 and 25, for instance.